### ISSUE TITLE
Models in CHAOS2019

### ISSUE THREAD

**Issue by mangoyuan** [MIC-DKFZ/nnUNet#84]
Thanks for your excellent sharing.
I notice that you paticipate in [CHAOS2019](https://chaos.grand-challenge.org/) and you use a ensemble of three 3D U-Nets. Do you mean that you use three internal variant type of nnU-Net and ensemble the total 15 models (3 model * 5 cross-validation)?
Did you try the original nnUNet in CHAOS2019?
Thanks.

**Comment by FabianIsensee** [comment]
Hi,
I use an internal variant of nnU-net which is a little better than the one here. Also I did manual modifications to one parameter (target spacing) as it was necessary for some cases to be perfect.
I submitted an ensemble of 5 3d_fullres models.
Best,
Fabian

**Comment by mangoyuan** [comment]
Thanks for your reply! Wish you all the best in your work.

**Comment by FabianIsensee** [comment]
I forgot to mention that task 3 & 5 require you to work on T1 in phase, T1 out phase and T2. I use the in phase and out phase image as separate training examples. For the test set submission I ensembled the results for in phase and out phase as only one mask was required

**Comment by mangoyuan** [comment]
Hi, Fabian.
I noticed it in the supplementary pdf. Thanks for reminding. =D
I used a variant nnUNet whose base_num_features=16 (limited GPU memory) and test it in Task5, CHAOS2019 using only fold0 best_model.ckpt. I found it performed pretty well (Dice score ~91% to ~94% yours), but not good in some cases, for example case35 which has two arms in MR Images.
Therefore, i am so curious about your internal variant nnU-net and tricks.
By the way, you give a very nice talk in MICCAI2019 which benefits me a lot.
Thanks all.
